android/java/org/chromium/chrome/browser/shields/BraveShieldsHandler.java

```diff
@@ -114,7 +122,7 @@ public BlockersInfo() {
     private View mHardwareButtonMenuAnchor;
     private final Map<Integer, BlockersInfo> mTabsStat =
         Collections.synchronizedMap(new HashMap<Integer, BlockersInfo>());
-
+    private Map<String, String> mDisconnectEntity = new HashMap<String, String>();
```

Comment:
Maybe rename `mDisconnectEntity` to `mDisconnectEntities` or `mResourceToCompanyName`
to more accurate reflect the meaning?